From f592d36ee4998e64eed28637ce6151dd90925347 Mon Sep 17 00:00:00 2001
From: Ziyue Yang <yzylivezh@hotmail.com>
Date: Thu, 27 Jul 2017 12:33:07 +0800
Subject: [PATCH] anti sandbox patch

---
 core/main.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/main.mk b/core/main.mk
index a6f829ab6..31e0a86aa 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -323,7 +323,7 @@ ifneq (,$(user_variant))
     tags_to_install += debug
   else
     # Disable debugging in plain user builds.
-    enable_target_debugging :=
+    # enable_target_debugging :=
   endif
 
   # Turn on Dalvik preoptimization for user builds, but only if not
-- 
2.13.2

