From deed667f35bfaa7c5934f04c2a22e234028dd3f3 Mon Sep 17 00:00:00 2001
From: Ziyue Yang <yzylivezh@hotmail.com>
Date: Tue, 25 Dec 2018 14:55:05 +0800
Subject: [PATCH] anti sandbox patch

---
 core/main.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/main.mk b/core/main.mk
index a6f829a..31e0a86 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -323,7 +323,7 @@ ifneq (,$(user_variant))
     tags_to_install += debug
   else
     # Disable debugging in plain user builds.
-    enable_target_debugging :=
+    # enable_target_debugging :=
   endif
 
   # Turn on Dalvik preoptimization for user builds, but only if not
-- 
2.14.1

