From 403c30e10e8f8f0e85896bf78417a495b5b30313 Mon Sep 17 00:00:00 2001
From: Ziyue Yang <yzylivezh@hotmail.com>
Date: Tue, 20 Jun 2017 13:16:41 +0800
Subject: [PATCH] anti sandbox patch

---
 core/main.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/main.mk b/core/main.mk
index a6f829ab6..31e0a86aa 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -323,7 +323,7 @@ ifneq (,$(user_variant))
     tags_to_install += debug
   else
     # Disable debugging in plain user builds.
-    enable_target_debugging :=
+    # enable_target_debugging :=
   endif
 
   # Turn on Dalvik preoptimization for user builds, but only if not
-- 
2.11.0

